src/Features/CSharp/Portable/ConvertToRecord/ConvertToRecordCommon.cs

```diff
@@ -499,109 +542,130 @@ private static SyntaxList<AttributeListSyntax> GetModifiedAttributeListsForPrope
+                            prevToken = token;
+                        }
+                        break;
+                    default:
+                        break;
                 }
-                return null;
             }
+            return null;
         }
 
         private static IEnumerable<XmlNodeSyntax> CreateParamComments(
-            ImmutableArray<PropertyDeclarationSyntax> properties,
+            ImmutableArray<PositionalParameterInfo> propertyResults,
             SyntaxTriviaList exteriorTrivia,
             LineFormattingOptions lineFormattingOptions)
         {
-            foreach (var property in properties)
+            foreach (var result in propertyResults)
             {
-                // get the documentation comment
-                var potentialDocComment = property.GetLeadingTrivia().FirstOrNull(trivia => trivia.IsDocComment());
-                var paramContent = ImmutableArray<XmlNodeSyntax>.Empty;
-                if (potentialDocComment?.GetStructure() is DocumentationCommentTriviaSyntax docComment)
-                {
-                    // get the summary node if there is one
-                    var summaryNode = docComment.Content.FirstOrDefault(node =>
-                        node is XmlElementSyntax element &&
-                        element.StartTag?.Name.LocalName.ValueText == DocumentationCommentXmlNames.SummaryElementName);
+                // add an extra line and space with the exterior trivia, so that our params start on the next line and each
+                // param goes on a new line with the continuation trivia
+                // when adding a new line, the continue flag adds a single line documentation trivia, but we don't necessarily want that
+                yield return SyntaxFactory.XmlText(
+                    SyntaxFactory.XmlTextNewLine(lineFormattingOptions.NewLine, continueXmlDocumentationComment: false),
+                    SyntaxFactory.XmlTextLiteral(" ").WithLeadingTrivia(exteriorTrivia));
 
-                    if (summaryNode != null)
+                if (result.IsInherited)
+                {
+                    // generate a param comment with an inherited doc pointing to the doc comment of the original
+                    // property. If there isn't one it will show up as blank. For prop "Example" from inherited record B:
+                    // <param name="Example"><inheritdoc cref="Base" path="/param[@name='Example']"/></param>
+                    yield return SyntaxFactory.XmlParamElement(result.Symbol.Name,
+                        SyntaxFactory.XmlEmptyElement(
+                            SyntaxFactory.XmlName(DocumentationCommentXmlNames.InheritdocElementName),
+                            SyntaxFactory.List(ImmutableArray.Create<XmlAttributeSyntax>(
+                                SyntaxFactory.XmlCrefAttribute(SyntaxFactory.TypeCref(
+                                    SyntaxFactory.ParseTypeName(result.Symbol.ContainingType.MetadataName))),
```

Comment:
Do we need to be doing any smarter qualification/simplification here? Does this also handle the different syntax for generics?

Reply:
See above, I realized we actually don't need the name reference or path attributes